sonaranalyzer-dotnet/build/rspec.ps1

```
@@ -242,5 +243,9 @@ if ($className -And $ruleKey) {
        UpdateTestEntry $csRuleData
     }
 
-    Remove-Item -Recurse -Force "${PSScriptRoot}\\..\\.vs"
```

Can't we just quietly ignore the error?